contrib/python/README.md

```diff
@@ -0,0 +1,11 @@
+Python contributions for ASPECT 
+==============================
+
+This folder contains some useful routines to read files with Python,
+using the numpy and pandas packages.
+
+To install Python with anaconda: 
+https://www.anaconda.com/distribution/
+
```

Comment:
Could you add a short description of how to start the ipython notebook?

Reply:
done!